packages/accounts/accounts-tab/client.js

```diff
@@ -4,23 +4,12 @@ ReactiveTemplates.onRendered('accounts.index', function() {
 })
 
 ReactiveTemplates.helpers('accounts.index', {
-  users: function () {
-    return Meteor.users.find({}, { sort: { createdAt: -1 } });
-  },
-  buttons: function() {
-    var self = this;
-    return _.filter(orion.accounts._adminUsersButtons, function(value, key, list){
-      if (typeof value.shouldShow != 'function') {
```

Comment:
I should probably just make a pull request directly to [orionjs/orion](https://github.com/orionjs/orion) (v1.1 branch) now that your PR has been merged. what do you think?